{{ cookiecutter.repo_name }}/requirements.txt

```diff
@@ -1,8 +1,9 @@
 Werkzeug==0.9.6
 bumpversion==0.5.1
-django-debug-toolbar==1.2.2
+django-debug-toolbar==1.3
 flake8==2.3.0
-https://github.com/dcramer/django-devserver/archive/master.zip
+https://github.com/transcode-de/django-devserver/archive/master.zip
 ipython==2.3.1
-pdbpp==0.8.2
+pdbpp==0.8.1
+https://github.com/bsvetchine/django-configurations/archive/master.zip
```

Comment:
Please explain why we are using a fork of django-configurations. It would be very helpful if you could add a link to a related issue.